pytorch_lightning/utilities/imports.py

```diff
@@ -54,4 +54,5 @@ def _module_available(module_path: str) -> bool:
 _GROUP_AVAILABLE = platform.system() != 'Windows' and _module_available('torch.distributed.group')
 _FAIRSCALE_PIPE_AVAILABLE = _FAIRSCALE_AVAILABLE and LooseVersion(torch.__version__) >= LooseVersion("1.6.0")
 _BOLTS_AVAILABLE = _module_available('pl_bolts')
+_PYTORCH_GREATER_EQUAL_1_6_0 = LooseVersion(torch.__version__) >= LooseVersion("1.6.0")
```

Comment:
why did we use `_module_available` for pruning but a version check for this?

Reply:
Because `from torch.optim.swa_utils import SWALR` was introduced in 1.6.0

Reply:
My point is, either pruning should use `_PYTORCH_GREATER_EQUAL_1_6_0`

Or SWA should use `_PYTORCH_SWA_AVAILABLE`

for consistency